README.md

```diff
@@ -54,21 +54,21 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 | macOS        | [64-bit][rl-macos-tar]                          | [Instructions][in-tar]   |
 | Linux        | [64-bit][rl-linux-tar]                          | [Instructions][in-tar]   |
```

Comment:
The binary archive chart needs to be updated with `linux-arm`, `win-arm` and `win-arm64` packages